composer.json

```diff
@@ -21,9 +21,9 @@
     "minimum-stability" : "dev",
     "require": {
         "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
-        "oat-sa/generis" : ">=15.22",
-        "oat-sa/tao-core" : ">=50.24.6",
-        "oat-sa/extension-tao-itemqti" : ">=29.13.0"
+        "oat-sa/generis" : ">=15.28.0",
+        "oat-sa/tao-core" : ">=53.7.3",
+        "oat-sa/extension-tao-itemqti" : ">=30.3.0"
```

Comment:
**thought:** I don't see the point of upgrading these dependencies. Does it add any value to the code? Does it bring something required by it? The only thing I see is that the extension is now tied to a more recent version of TAO while it could perfectly work with an older version... This PR is related to a PCI, which is standalone per design and should not depend on the host.

**issue:** Keeping with this upgrade means that a useless backport will be needed to make the PCI available to an older version.